test/integration/overload_integration_test.cc

```diff
@@ -5,6 +5,8 @@
 #include "envoy/server/resource_monitor.h"
 #include "envoy/server/resource_monitor_config.h"
 
+#include "source/extensions/resource_monitors/downstream_connections/downstream_connections_monitor.h"
```

Comment:
It would be best to move tests that depend on this extension to a new test/extensions file so it is possible to build envoy core without pulling in this extension.